node/src/Pos/Block/Worker.hs

```diff
@@ -97,11 +97,18 @@ blkOnNewSlot =
 -- Block creation worker
 ----------------------------------------------------------------------------
 
-blockCreator
-    :: WorkMode ssc ctx m
-    => SlotId -> SendActions m -> m ()
+blockCreator ::
+       forall ssc ctx m. WorkMode ssc ctx m
+    => SlotId
+    -> SendActions m
+    -> m ()
 blockCreator (slotId@SlotId {..}) sendActions = do
 
+    -- Print difference between tip slot and current slot.
+    -- TODO: probably not the best place, but it's not the biggest concern now.
+    tipHeader <- DB.getTipHeader @ssc
```

Comment:
My suggestion had been to move this into the where clause too, to show that tipHeader is not used anywhere else later in the scope.

Reply:
Ah, sorry @dcoutts , I understood something different, my bad 🙀